UI/accounts.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Admin's page</title>
+	<link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+<body>
+<header> 
+  <div>
+  
+  <ul>
+  <a href="admin_index.html"><img  src = "img/logo.png" style="float: left;position: relative;height:10px ;height :45px "></a>
+  <li><a href="admin_index.html">Products</a></li>
+  <li><a class="active" href="#">Accounts</a></li>
+  <li><a href="sales.html">Sales</a></li>
+  <li style="float:right"><a href="login.html">Log out</a></li>
```

Comment:
avoid inline css. add it to your external css file

Reply:
@nuwagalb thanx i will correct it